browser.js

```diff
@@ -14,11 +14,14 @@ if (crypto && crypto.getRandomValues) {
 
 function randomBytes (size, cb) {
   // in browserify, this is becomes an extended UInt8Array
-  var bytes = new Buffer(size)
+  var rawBytes = new global.UInt8Array(size)
 
   // This will not work in older browsers.
   // See https://developer.mozilla.org/en-US/docs/Web/API/window.crypto.getRandomValues
-  crypto.getRandomValues(bytes)
+  crypto.getRandomValues(rawBytes)
```

Comment:
so, my understanding is `phantomjs` didn't like giving `crypto.getRandomValues` a `Buffer` (aka, extended `UInt8Array`).
But, it is OK with initializing a `Buffer` with a `UInt8Array`.

Right?